src/main/java/io/jenkins/plugins/signpath/SignPathStepBase.java

```diff
@@ -38,29 +38,16 @@ public abstract class SignPathStepBase extends Step {
     // also a timeout that is too high is not useful anymore
     private int waitForPowerShellTimeoutInSeconds = (int) TimeUnit.MINUTES.toSeconds(30);
 
-    private String trustedBuildSystemTokenCredentialId;
     private String apiTokenCredentialId = "SignPath.ApiToken";
 
     public String getApiUrl() {
-        SignPathPluginGlobalConfiguration config = GlobalConfiguration.all().get(SignPathPluginGlobalConfiguration.class);
-        if(config != null) {
-            return config.getApiURL();
-        }
-        else {
-            return null;
-        }
+        return getSignPathConfig().getApiURL();
```

Comment:
Just out of curiosity: Why are there these wrapper getter methods in the StepBase class?

Reply:
To encapsulate the retrieval of SignPathPluginGlobalConfiguration from the configuration registry, we need a null check there. To avoid duplicating it everywhere, I added a wrapper.

Reply:
@volodymyr-bobko : Yes, I get that the method `getSignPathConfig()` with the null check makes sense. But `getApiUrl()` as a wrapper is barely used. Okay for me to keep it, just wanted to know if that was a deliberate design decision or just a leftover